src/api/getTransactionHashByExecutionId.ts

```diff
@@ -0,0 +1,13 @@
+import { Repository } from 'typeorm'
+import { EExecutionState, ScheduledExecution } from '../entities'
+
+const getTransactionHashByExecutionId = async (
+  repository: Repository<ScheduledExecution>,
+  executionId: string): Promise<string | undefined> => {
+  const result = await repository.findOne(executionId)
+  if (result && result.state === EExecutionState.ExecutionSuccessful) {
```

Comment:
Isn’t it more robust to return the status and reason? I would also add the id to the response